### ISSUE TITLE
a warning from grep about missing file

### ISSUE THREAD

**Issue by DanGrayson** [UniMath/UniMath#1322]
... results from the line in `Makefile`:
```
ifeq ($(shell grep -q ^VDFILE build/CoqMakefile.make && echo yes),yes)
```
, which is run before the file even exists.  Think of a better way to figure out what files
contain the dependencies.  Or start assuming Coq is more modern.

**Comment by benediktahrens** [comment]
What is the status of this issue, @DanGrayson ?

**Comment by DanGrayson** [comment]
Thanks for the reminder, I've just fixed it in commit bd2508582f914fc4a44261aa188f34e45c646cc3
